recipes/powershell/assets/README.md

```diff
@@ -0,0 +1,23 @@
+NetBackup Assets Service API Code Samples

```

Comment:
Please use markdown formatting in this file. You can start with level-1 heading here.

```suggestion
# NetBackup Assets Service API Code Samples
```